### DISCUSSION TITLE
Teleport Connect 2022 - Nov 9th

### DISCUSSION THREAD

**Discussion by benarent** [gravitational/teleport#16533] [General]
## About Teleport Connect 2022
Teleport Connect is a single-day dual-track user conference. This event is a chance for the Teleport community to get together and discuss the industry challenges surrounding Access, DevSecOps, and deploying secure systems at scale.

Hosted at the historic San Francisco Mint, this is a unique opportunity to learn, network, and collaborate with others through a plethora of presentations, panels, office hours, demos and more!

After a day of presentations, we'll retreat to the historic Vaults for a speakeasy happy hour.

### Get your ticket
https://goteleport.com/teleport-connect-2022/

### Teleport Connect 2022 - Agenda
<table>
  <tr>
   <td>8:00 - 9:00 
   </td>
   <td colspan="2" >Breakfast
   </td>
  </tr>
  <tr>
   <td>9:00 - 9:10
   </td>
   <td colspan="2" >Opening Remarks
   </td>
  </tr>
  <tr>
   <td>9:15 - 9:50
   </td>
   <td colspan="2" >Special External Keynote [TBA]
   </td>
  </tr>
  <tr>
   <td>9:50 - 10:15
   </td>
   <td colspan="2" >Teleport Special Announcement by Ev Kontsevoy, Teleport CEO
   </td>
  </tr>
  <tr>
   <td>10:15  
   </td>
   <td>Break
   </td>
   <td>
   </td>
  </tr>
  <tr>
   <td>10:30 - 11:00
   </td>
   <td>Embark Trucks: Koté Mushegian. Connecting Self-Driving Semi Trucks. 
<p>
<i>Security + Identity + Policy</i> - <i>Server / IoT</i>
   </td>
   <td>Sage Science: Stefan Sedich. Teleport + JupyterLab: simplifying access in a cloud-native world 
<p>
<i>Productivity</i> - <i>Kubernetes</i>
   </td>
  </tr>
  <tr>
   <td>11:00 - 11:30
   </td>
   <td>Session [TBA]
   </td>
   <td>Session [TBA]
   </td>
  </tr>
  <tr>
   <td>11:30 - 12:00
   </td>
   <td>IBM Instana: Hunter Madison. Managing Dropwizard with Teleport 
<p>
<i>Customizing & Extending Teleport -  DevOps </i>
   </td>
   <td>BuyersEdgePlatform: Josh Wieder. HA Teleport in Multi-Account AWS
<p>
Operators  DevOps 
   </td>
  </tr>
  <tr>
   <td>12:00 - 12:30
   </td>
   <td>Teleport: Gus:  "Huh, I didn't know Teleport could do that"
<p>
<i>Customizing & Extending Teleport - DevOps</i>
   </td>
   <td>Qwilt: Tomer Tcherniak: Teleport Qwilt Design
<p>
<i>Customizing & Extending Teleport - DevOps</i>
<p>
 
   </td>
  </tr>
  <tr>
   <td>12:30 - 13:30
   </td>
   <td colspan="2" >Lunch
   </td>
  </tr>
  <tr>
   <td>13:00 - 13:45
   </td>
   <td>Panel: Access at Scale  [ Cari Life Sciences, TBA, TBA ]
<p>
Operators  DevOps 
   </td>
   <td>Panel:  [ Reed From Teleport.  & Sendbird, TBA, TBA ] 
<p>
Security 
   </td>
  </tr>
  <tr>
   <td>13:45 - 2:15 
   </td>
   <td>Dropbox: Jussi Lundstedt.  Deploying Teleport at Scale with Custom Infrastructure
<p>
<i>Operators  -  Server / IoT</i>
   </td>
   <td>Figment: Annalea Ilg. "Eliminating Security Overhead"
<p>
Security 
   </td>
  </tr>
  <tr>
   <td>2:15 - 2:30 
   </td>
   <td colspan="2" >Break
   </td>
  </tr>
  <tr>
   <td>2:30 - 3:30
   </td>
   <td>Worldcoin: Ethan Wright. (mis)Using Teleport as a Fleet Management Tool
<p>
<i>Customizing & Extending Teleport - Server / IoT</i>
<p>
 
   </td>
   <td>Jasper.ai: John Baird. Ephemeral Kubernetes Environments
<p>
<i>Customizing & Extending Teleport - Kubernetes </i>
   </td>
  </tr>
  <tr>
   <td>3:30 - 4:00
   </td>
   <td>Moody’s: Matthew Davis. Deploying in the fray
<p>
<i>Operators - DevSecOps</i>
   </td>
   <td>Vonage: Eddie Wassef. "How we extended Teleport seamlessly to our 100's of clusters"
<p>
<i>Customizing & Extending Teleport - Kubernetes</i>
   </td>
  </tr>
  <tr>
   <td>4:00 - 4:30
   </td>
   <td>FedRAMP:  Fireside Chat [TBA]
<p>
Operators  
   </td>
   <td>Session [TBA]
   </td>
  </tr>
  <tr>
   <td>4:30 - 4:45
   </td>
   <td>Break
   </td>
   <td>
   </td>
  </tr>
  <tr>
   <td>4:45 - 5:00
   </td>
   <td colspan="2" >Closing Thoughts 
   </td>
  </tr>
  <tr>
   <td>5:00 - 7:00
   </td>
   <td colspan="2" >Speakeasy Happy Hour & Heavy Hors d'oeuvres.
   </td>
  </tr>
</table>

**Comment by benarent** [comment]
Our final agenda is now available on the Teleport Connect 2022 website. https://goteleport.com/teleport-connect-2022/

**Comment by jackletter** [comment]
收到了 哈哈
